core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/XMLReaderBatchSource.java

```diff
@@ -248,11 +248,15 @@ public void onRunFinish(boolean succeeded, BatchSourceContext context) {
       "Example - For tableExpiryPeriod = 30, data before 30 days get deleted from the table.")
     private final String tableExpiryPeriod;
 
+    @Description("Existing hdfs folder path having read and write access to the current User, required for internal" +
```

Comment:
space after "internal". Also, will need to explain it in more detail than "internal computation of the plugin", since you're exposing this to the user now. Specify the exact purpose of this directory